app/components/subcases/side-nav.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class SubcaseSideNavComponent extends Component {
+  @service intl;
+
+  @action
+  async calculateDisplayLabel(subcaseId) {
+    const subcases = this.args.subcases.slice();
+    const subcase = subcases.find((subcase) => subcase.id === subcaseId);
+    const number = subcases.indexOf(subcase);
+    const subcaseType = await subcase.type;
+    if (subcaseType) {
+      const label = await subcaseType.label;
+      return label;
+    } else {
+      return this.intl.t("step-and-number", {number: number+1})
+    }
+  }
```

Comment:
I'm assuming it's because of the index calculation that needs access to the `subcases` list. Might change the `Subcases::SideNav::Subcase` to accept an index as well.

Reply:
Fixed in [e3e52d0](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/2041/commits/e3e52d0870926eb57316ebcf005d4594c6966166)